### PR TITLE
Fix fmt & remaining md links

### PR DESCRIPTION
**Pull request by rmalmain**

### CODE REVIEW

File: .github/workflows/md-config.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
       "pattern": "^https://crates.io"
     }
   ],
-  "aliveStatusCodes": [0, 200]
+  "aliveStatusCodes": [0, 200, 403]
```
Comment (domenukk) [inline .github/workflows/md-config.json:7]: What's that for? Seems wrong? :D

Comment (rmalmain) [reply .github/workflows/md-config.json:7]: it is.
this link returns a 403 and i was too lazy to look more closely: https://www.intel.com/content/www/us/en/support/articles/000056730/processors.html

Comment (rmalmain) [reply .github/workflows/md-config.json:7]: https://github.com/AFLplusplus/LibAFL/actions/runs/10517109988/job/29140616715#step:4:640

Comment (domenukk) [reply .github/workflows/md-config.json:7]: lol. Welp..

I guess they forbid curl access or something like that

**Pull request merged**